frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -26,14 +26,19 @@ import './_vm-resource.scss';
 
 export const VMDetailsItem: React.FC<VMDetailsItemProps> = ({
   title,
+  canEdit = false,
+  editButtonId,
+  onEditClick,
   idValue,
   isNotAvail = false,
   valueClassName,
   children,
 }) => {
   return (
     <>
-      <dt>{title}</dt>
+      <dt>
+        {title} <EditButton id={editButtonId} canEdit={canEdit} onClick={onEditClick} />
```

Comment:
I have commented this in another PR.
The edit button (pencil) should go to the value to stay consistent.

Reply:
We have one Modal to edit multiple CDs so its logical to have one button for it and not per cd row.

@matthewcarleton thoughts?

Reply:
@mareklibra @glekner we actually have a design that is moving more towards having a single edit action for the list
https://docs.google.com/document/d/15aOWgpkd0Zp0hvbKM4jfG6Jf00FHjY_f92bkosbo1Ds/edit#heading=h.vqo42z9akrnk
So I'd so long term we should see this edit icon next to the title and not next to each item. That being said, we want things to stay consistent so this should be a console wide change where all details views offer the same edit approach. 
I'd rather see one edit action for CD-ROMs specifically b/c of the nature of editing them. So maybe it's ok to have it different in this case, but it would be great to know when we could see the update happen across the entire console too :)